pop-api/integration-tests/contracts/create_token_in_constructor/Cargo.toml

```diff
@@ -3,6 +3,8 @@ edition = "2021"
 name = "create_token_in_constructor"
 version = "0.1.0"
 
+[workspace]
```

Comment:
Excludes contract from workspace.

Reply:
Nice workaround, but might it be better to set in the workspace root via so that anything within contracts is automatically excluded:

```
[workspace]
exclude = ["pop-api/integration-tests/contracts/*"]
```

https://doc.rust-lang.org/cargo/reference/workspaces.html#the-members-and-exclude-fields

Note: I dont know that this works, just seems a more robust way as its defined in a single place and automatically excludes any future contracts added.

Reply:
adding this to INCLUDE does work: `"pop-api/integration-tests/contracts/*",
`

Reply:
I tried adding to the exclude and the include. Exclude the same problem arises (crate thinks it's in a workspace), while including it results in the build hanging.

Reply:
Nevermind, tried again and it worked by including. Weird.